recorder/src/main/cpp/agent.cc

```diff
@@ -130,7 +130,13 @@ static bool PrepareJvmti(jvmtiEnv *jvmti) {
         JVMTI_ERROR_CLEANUP_RET(
             jvmti->AddCapabilities(&caps),
             false,
-            logger->error("Failed to add capabilities with error {}", error))
+            logger->error("Failed to add capabilities with error {}", error));
+    }
+    if (jvmti->AddToSystemClassLoaderSearch(CONFIGURATION->pctx_jar_path) != JVMTI_ERROR_NONE) {
+        logger->error("Failed to add path to perf-ctx jar ({}) to classpath {}", CONFIGURATION->pctx_jar_path, error);
+        return false;
+    } else {
+        logger->debug("Added perf-ctx jar '{}' to classpath", CONFIGURATION->pctx_jar_path);
```

Comment:
Maybe logger->info instead? Even if debug mode is disabled, for any error that pops up, knowing which perf-ctx jar was actually loaded will help.
Actually logging(info level) CONFIGURATION object makes sense at process start(if we are not doing it already). If jvm has not crashed, someone can look at running process info to find launch args, but if it has, logs surfacing this becomes important